kcards/tests/test_models_room.py

```diff
@@ -90,84 +143,28 @@ def it_removes_the_green_speaker_if_only_greens_are_queued(room):
             ]
 
         def it_clears_the_queue_without_active_discussion(room):
-            room.add("John Doe", 'green')
-            room.add("Dan Lindeman", 'green')
+            room.add_card("John Doe", 'green')
+            room.add_card("Dan Lindeman", 'green')
             room.next_speaker()
             room.next_speaker()
             expect(room.queue) == []
 
         def it_clears_the_queue_with_active_discussion(room):
-            room.add("John Doe", 'green')
-            room.add("Jace Browning", 'yellow')
-            room.add("Dan Lindeman", 'green')
+            room.add_card("John Doe", 'green')
+            room.add_card("Jace Browning", 'yellow')
+            room.add_card("Dan Lindeman", 'green')
             room.next_speaker()
             room.next_speaker()
             room.next_speaker()
             expect(room.queue) == []
 
         def it_removes_the_red_speaker(room):
-            room.add("John Doe", 'green')
-            room.add("Jace Browning", 'red')
+            room.add_card("John Doe", 'green')
+            room.add_card("Jace Browning", 'red')
             room.next_speaker()
             expect(room.queue) == [
                 {
                     'name': "John Doe",
                     'color': 'green',
                 }
             ]
-
-    def describe_queue():
-
-        def when_empty(room):
```

Comment:
I decided those were actually tests of the `add_card` method so I renamed them and added some more dedicated tests.